src/Klsandbox/BillplzRoute/Services/BillplzResponseManager.php

```diff
@@ -52,7 +52,7 @@ public function getBill($bill_id)
         return $bill;
     }
 
-    public function createBill($data)
+    public function createBill($data, $billplzKey)
```

Comment:
collection_id is the part of $data, i send collection_id on controller and send you pull request on bitbucket.